gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -1992,15 +1993,19 @@ The above example retrieves the character's name typed into the TextEdit control
 \n\
   GemRB.SetToken ('VoiceSet', TextAreaControl.QueryText ())\n\
 The above example sets the VoiceSet token to the value of the selected string in a TextArea control. Later this voiceset could be stored in the character sheet.\n\
+If returned string will be used to e.g. as filename, string should be using system encoding, or file might be not found if it contains non-ASCII letters.\n\
+**Example**\n\
+  Filename = CharImportList.QueryText (True)\n\
+  GemRB.CreatePlayer (Filename, MyChar|0x8000, 1)\n\
 \n\
 **See also:** [[guiscript:Control_SetText]], [[guiscript:SetToken]], [[guiscript:accessing_gui_controls]]"
 );
 
 static PyObject* GemRB_Control_QueryText(PyObject * /*self*/, PyObject* args)
 {
-	int wi, ci;
+	int wi, ci, useSystemEncoding = 0;
 
-	if (!PyArg_ParseTuple( args, "ii", &wi, &ci )) {
+	if (!PyArg_ParseTuple( args, "ii|i", &wi, &ci, &useSystemEncoding )) {
```

Comment:
you can use `p` as a format specifier for booleans.

Reply:
I was initially trying to do so, but it was introduced only in python 3.3, I believe.

Reply:
Yes, here is the changelog entry. https://hg.python.org/cpython/file/05274ab06182/Misc/NEWS